### DISCUSSION TITLE
How to use Tesla proxy add-on with duckdns+nginx configuration

### DISCUSSION THREAD

**Discussion by paky79** [llamafilm/tesla-http-proxy-addon#16] [General]
Hello, I am currently exposing my HA with the standard configuration duckdns+nginx add-ons. 

Using duckdns seems to be impossible to allow Tesla to activate the developer profile(the procedure fails immediately due to allowed URL maybe because of .duckdns.org). 

I am wondering how can fix the issue: will I buy a .com DNS and use cloudlfare tunnel? Reading your guidelines seems that I need anyway to use NGINX to reverse proxy, so how can configure that? Or should I buy a .com DNS, create SSL certificates and configure NGINX?

I think this could be very useful to understand to all those people using my same standard configuration and not so expert in network configuration.

**Comment by baylanger** [comment]
DuckDNS is reported to work at least by 1 user. See this [discussion](https://github.com/llamafilm/tesla-http-proxy-addon/discussions/14) it might help you out.
